omnibus/config/software/libbcc.rb

```diff
@@ -0,0 +1,14 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-2020 Datadog, Inc.
+
+name 'bcc'
+
+dependency 'libelf'
+
+build do
+  command "#{ENV['S3_CP_CMD']} #{ENV['S3_ARTIFACTS_URI']}/libbcc-#{ENV['PACKAGE_ARCH']}.tar.xz /tmp/libbcc.tar.xz"
```

Comment:
What would happen if a user wants to build the system-probe (ie: outside the CI)? I think it will fail because `S3_ARTIFACTS_URI` is not defined?

Maybe we can treat this new dependency the same way we treat `jmxfetch`, for which we specify a pre-built version here [1] and download it in omnibus. This way, if a user does a custom build, it will still pick up whatever version is specified. Plus, it will be re-built only when there are changes.

[1] https://github.com/DataDog/datadog-agent/blob/master/release.json#L6

Reply:
Actually... is there any good reason not to treat the entire system-probe this way, like we do with jmxfetch? @olivielpeau @sunhay What's the benefit of having it live in the same repo?

Reply:
We used to have it this way, and merged it when the process-agent and trace-agents were being merged into the single repo.

Having them in the same repo has some advantages:
- releasing is easier as the process for pulling a new system-probe into the agent was (tag system-probe >   agent PR to pull new version in > tag agent) 
-  dependency management is easier since the system-probe pulls in some agent packages (mostly utilities and stuff related to config)

Reply:
It makes sense if it depends on shared files.

Reply:
The core agent and system-probe are, in the end, packaged together in a single package/docker image. So, they share the same lifecycle.
In a followup PR (#4619), I introduce communication between the two.
The fact that they are both hosted in the same repository ease that work since the data structures that are exchanged between the two can be defined at only one place.